ui/admin/tests/e2e/helpers/boundary-ui.js

```diff
@@ -287,3 +287,121 @@ exports.addBrokeredCredentialsToTarget = async (
   await page.getByRole('button', { name: 'Dismiss' }).click();
   await expect(page.getByRole('link', { name: credentialName })).toBeVisible();
 };
+
+/**
+ * Uses the UI to create new Auth Method. Assumes you have selected the desired scope.
+ * @param {Page} page Playwright page object
+ * @param {string} authMethodName Name of new auth method
+ */
+exports.createNewAuthMethod = async (page, authMethodName) => {
```

Comment:
None blocking question: I am curious how we specify the type of auth method we are creating? I can't see were we are specifyng auth method type. I can see on line 301 & 302 of this same file we are selecting the type password. Would it be possible type is a parameter of this method?

Reply:
>  I am curious how we specify the type of auth method we are creating?

It's hardcoded:
`await page.getByText('Password', { exact: true }).click();`

It's possible to add the parameter. However in the method we'd have two big branches: one for password method and the other for the OIDC as they have different set of required fields. 
Now this method is used for creating password auth method only. So, I'd propose sticking to the current implementation and rename a method to make it clear what auth method type it creates

```suggestion
exports.createNewPasswordAuthMethod = async (page, authMethodName) => {
```

What do you think @calcaide?

Reply:
@stasryzhov I think you explained very clear and I agree with you 😃 Thanks for clarification and modification, appreciate it!!